pkg/cmd/extension/manager.go

```diff
@@ -81,7 +83,9 @@ func (m *Manager) Dispatch(args []string, stdin io.Reader, stdout, stderr io.Wri
 
 	var externalCmd *exec.Cmd
 
-	if runtime.GOOS == "windows" {
+	if ext.IsBinary() {
```

Comment:
```suggestion
	if ext.IsBinary() || runtime.GOOS != "windows" {
	    ...
	} else {
	    ...
	}
```
Looks like this if statement could be simplified a bit.

Reply:
lol oops